Tests/KituraOpenAPITests/KituraTestBuilder.swift

```diff
@@ -0,0 +1,299 @@
+/**
```

Comment:
Ugh do we really have to copy and paste all this?

Reply:
The alternative is to make KituraTest and KituraTestBuilder part of a separate project which is imported as a test dependency.

Neil and I discussed this, but because SPM does not have 'test dependencies', it would have the side-effect of introducing a common dependency to Kitura and (potentially) many other repos that depend on Kitura.  Unlike other common dependencies (eg. LoggerAPI), we'd probably find ourselves needing to change the base Kitura test classes relatively frequently - eg. to add new functionality - which I anticipate would cause headaches for keeping our packages in a resolvable state.